tools/wpt/expectation.json

```diff
@@ -0,0 +1,610 @@
+{
+  "WebCryptoAPI": {
+    "getRandomValues.any.js": true
+  },
```

Comment:
WebIDL test cases might be dropped here?

`wpt.jsonc` had the following:
```
  "WebIDL": [	
    "ecmascript-binding/es-exceptions/DOMException-constants",	
    "ecmascript-binding/es-exceptions/DOMException-constructor-and-prototype",	
    "ecmascript-binding/es-exceptions/DOMException-constructor-behavior",	
    {	
      "name": "ecmascript-binding/es-exceptions/DOMException-custom-bindings",	
      "expectFail": [	
        // TODO(kt3k): Enable this test.	
        // We can pass this test by using Object.setPrototypeOf(...) instead of	
        // class...extends, but that causes a problem in printing of uncaught	
        // DOMException. We might need to modify how to print uncaught error in	
        // `//core/error.rs`.	
        "does not inherit from Error: class-side"	
      ]	
    }	
  ]
```